src/controllers/class.js

```diff
@@ -7,7 +7,7 @@ const getClasses = (req, res) => {
     .then((data) => {
       if (data) {
         res.status(200).json({
-          message: 'All classes',
+          message: 'Classes list',
           data,
         });
       }
```

Comment:
you should add a property "data" and "error", the response it should be message: 'classes list', data: data, error: false

Reply:
The ticket only required the changes on messages, the structure of the http response will be changed in another ticket.